docs/user-guide/configure.md

```diff
@@ -252,6 +252,10 @@ In your `.stylelintrc.json`:
     "alpha-value-notation": "number"
   },
   "overrides": [
+    {
+      "files": ["*.scss", "**/*.scss"],
```

Comment:
[ask] Are these patterns the same as below?

```suggestion
      "files": ["**/*.scss"],
```

Reply:
I don't think that pattern will match files in the root.

See: 
https://github.com/ota-meshi/stylelint-config-html/blob/af14c2bfa55eab72461f7047c25cc4f842b19996/html.js#L23

Reply:
I got it. 👍🏼